src/entity/service.ts

```diff
@@ -81,6 +84,14 @@ async function loadEntities(
 
   for (const e of entityDtos) {
     logger.info(`Importing entity ${e.id}`, e)
+
+    for (const did of e.dids) {
+      const didDocument = await didResolver.resolveDid(did)
+      if (!didDocument) {
+        throw new Error(`DID ${did} is not resolvable`)
+      }
+    }
```

Comment:
Does this work as intended? We don't want to load _any_ of the entities from the static json if one of the DIDs fails to resolve?

Reply:
Kind of, we don't want to load any entity after this happens, in other words, we're loading entities but when the first fail we stop, therefore we loaded only the entities before the one that failed.

Reply:
I was struggling with it a bit. First, I didn't in a way that we load every entity that's correct and then return just error report for the ones that failed. Then I realized it might be quite hard to update the file to just load the correct ones, or understand that you can load everything again with fixed entries and it will just overwrite everything. 🤷‍♂️  Maybe it's a good feedback that we should move towards a solution with update during runtime on during deploy.

Reply:
If we're throwing an error, the trust-registry will fail to start won't it?
I don't see any place where the error is handled

Reply:
Yes, it won't start. That's expected.